### PR TITLE
Upgraded example projects

### PR DESCRIPTION
**Pull request by nosmirck**
- Latest Packages
- Latest Android and iOS templates
- Added Web
- Fixed minor issues with latest go_router (deprecated .location)

### CODE REVIEW

**Comment by lucavenir** [comment]
Wow, kudos for the effort. I did *not* expect a PR on this example repo.

Let me check the changes and I'll come back to you

File: example/lib/utils/state_logger.dart [hunk 1]
```diff
@@ -13,7 +14,7 @@ class StateLogger extends ProviderObserver {
     Object? newValue,
     ProviderContainer container,
   ) {
-    print('''
+    debugPrint('''
```
Comment (lucavenir) [inline example/lib/utils/state_logger.dart:17]: Did you know? `debugPrint` [still prints in production](https://api.flutter.dev/flutter/foundation/debugPrint.html) 😄

Maybe you wanted to throttle? Anyways, It doesn't really matter for the sake of this example.

**Review by lucavenir — CHANGES_REQUESTED** [review]
Thank you again for this PR. I'm inclined to approve the changes ASAP, just please remove the `.DS` files, thanks

File: example/lib/utils/state_logger.dart (hunk 1 above)
Comment (nosmirck) [reply example/lib/utils/state_logger.dart:17]: I was just fixing a lint for that

File: .DS_Store
[the remark this answers is not in the document]
Comment (nosmirck) [reply .DS_Store:1]: removed :)

**Review by lucavenir — APPROVED** [review]

**Comment by lucavenir** [comment]
Thank you again for showing love to this ❤️

**Pull request merged**